features/270-next-skycli/commands.md

```diff
@@ -257,6 +257,32 @@ $ skycli app view-user-config > ./user-config.yaml
 $ skycli app update-user-config -f ./user-config.yaml
 ```
 
+### skycli app get-credentials
+
+`skycli app get-credentials`
```

Comment:
I think the subcommand name is unclear: what are the credentials? Something like `get-k8s-credentials` would be clearer.